lib/trento/clusters.ex

```diff
@@ -225,4 +230,23 @@ defmodule Trento.Clusters do
   end
 
   defp maybe_request_checks_execution(error), do: error
+
+  @spec get_ensa_version([String.t()]) :: EnsaVersion.t()
+  defp get_ensa_version(sap_system_ids) do
```

Comment:
The name is a bit misleading for me, but I don't have any better suggestion, so...

Reply:
As far as I know we don't really have terminology for this. The best I can think of is `get_aggregated_ensa_version()`, what do you think?